website/docs/r/load_balancer.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: "cloudflare"
+page_title: "Cloudflare: cloudflare_load_balancer"
+sidebar_current: "docs-cloudflare-resource-load-balancer"
+description: |-
+  Provides a Cloudflare Load Balancer resource.
+---
+
+# cloudflare_load_balancer
+
+Provides a Cloudflare Load Balancer resource. This sits in front of a number of defined pools of origins and provides various options for geographically-aware load balancing.
+
+## Example Usage
+
+```hcl
+resource "cloudflare_load_balancer_pool" "foo" {
+  name = "example-lb-pool"
+  origins {
+    name = "example-1"
+    address = "192.0.2.1"
+    enabled = false
+  }
+}
+
+# Define a load balancer which always points to the pool we defined
+# In normal usage, would have different pools set for different pops/regions
+# And some failover defined within that
+resource "cloudflare_load_balancer" "bar" {
```

Comment:
Nitpick: let's move the `cloudflare_load_balancer` config to the top of the `hcl` example, so it's the first thing users see. Otherwise, the user is learning about `cloudflare_load_balancer` and the first thing they see is `cloudflare_load_balancer_pool` configuration.

Reply:
>different pools set for different pops/regions

Also, what's a `pop`? It's best to not assume users are familiar with acronyms specific to a service. I'm not familiar with load balancing in detail, so I can't tell if `pop` and `region` are synonymous, or two distinct acceptable options.

Reply:
I tried to apply this example config and got similar error to the test failures